docs/GeoReplication.md

```diff
@@ -0,0 +1,137 @@
+
+# Pulsar Geo-Replication
+
+<!-- TOC depthFrom:2 depthTo:3 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Replication model](#replication-model)
+- [Configuring replication](#configuring-replication)
+	- [Granting permissions to properties](#granting-permissions-to-properties)
+	- [Creating the namespace](#creating-the-namespace)
+	- [Using global topics](#using-global-topics)
+
+<!-- /TOC -->
+
+## Replication model
+
+Pulsar properties provisioned on more than one cluster can enable replication
+among those clusters.  Replication is managed directly by the users, at the
+namespace level. Any topic in a global namespace is replicated. A user creates a
+global namespace, and configures that namespace to replicate between two ore more
+of the provisioned clusters. Any message published on any topic in that namespace
+will then be replicated to all clusters in that set.
+
+Messages are first stored in the local cluster, and then forwarded
+asynchronously to the remote clusters.
+
+In normal case, with no connectivity problems, messages are replicated immediately,
+at the same time as they are dispatched to local consumers. Typically, end-to-end
+delivery latency is defined by the network *RTT* between the remote regions.
+
+Applications can create producers and consumers in each of the clusters,
+even when the remote clusters are not reachable (eg: during a network
+partition).
+
+Subscriptions are local to the clusters in which they are created and they
+cannot be transferred between clusters.
+
+![Replication Diagram](img/GeoReplication.png)
+
+In the above example, the topic is being replicated between 3 clusters,
+***Cluster-A***, ***Cluster-B*** and ***Cluster-C***.
+
+All messages produced in any cluster will be delivered to all subscriptions
+in all the clusters. In this case, both ***C1*** and ***C2*** will receive all
+messages published by ***P1***, ***P2*** and ***P3***. Ordering is still
+guaranteed per-producer.
+
+## Configuring replication
+
+### Granting permissions to properties
+
+When each tenant is on-boarded to a Pulsar cluster, it needs to be
+granted permissions to be able to create namespaces in any particular cluster.
+
+When creating a new property, specify all the intended clusters:
```

Comment:
~~When each tenant is on-boarded to a Pulsar cluster, it needs to be~~
~~granted permissions to be able to create namespaces in any particular cluster.~~

~~When creating a new property, specify all the intended clusters: ~~

To replicate to a cluster, the tenant needs permission to use that cluster.  To grant this  permissions at  creation time,  specify all the intended clusters when creating the property